TEAM-POLICIES.md

```diff
@@ -0,0 +1,55 @@
+# Team Policies
+
+This documents two policies, 1) observing a team meeting, and 2) joining the team.
+
+## Team composition
+
+This document aims to formalize the procedures for getting membership in the Rust
+community team.
+
+------
+
+In general, we should encourage participation from community members who are not part of the team,
+as outlined in the observer policy (below).
+
+In some cases such participating members may have enough involvement that we feel it advatageous to
+include them in the team. Some reasons for doing this may be:
+
+ - Recognition: If someone is especially active in this space, it is worth adding them to the team to recognize them.
+ - "Officializing": For some tasks, like talking with other companies, it is useful to be able to say you are "X of the Rust Community Team"
+
+Most community team activities should be designed such that as much as possible, team membership is not required.
+
+To add someone as a member, they need to be nominated by an existing member on the private community team mailing list.
```

Comment:
Should this be crates.io team not community team?

Reply:
Also the reference to the mailing list, which we don't currently have.